### ISSUE TITLE
plugin quadrotor for Gazebo: vertical bias

### ISSUE THREAD

**Issue by jmplaza** [RoboticsURJC/JdeRobot#217]
The drone goes slowly upwards and it should not.

**Comment by varhub** [comment]
Problem should be located at:
```
QuadrotorPlugin::OnUpdate()
    (navi_state == Flying) ---> link->AddRelativeForce(force);
```
And force.z must be 0 or counter gravity. Related to:
```
    force.z = mass * (controllers_.velocity_z.update(
        velocity_command_->linearZ,
        velocity.z, acceleration.z, dt) +
        load_factor * gravity);
```

Paranoid check: ensure that `velocity.z` and `acceleration.z` are **zero** when *stop*

**Comment by varhub** [comment]
"Fixed" it after tune velocityZ PID parameters.
This was issue before and after.
Accurate fix (takeoff) requires **fix_position** commands (go to XYZ)

**Comment by jmplaza** [comment]
Solved with solution to #229
